db/seeds/dev/favorites.js

```diff
@@ -0,0 +1,19 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('favorites').del()
+      // Inserts seed entries
+    .then(() => {
+      return Promise.all([
+        knex('favorites').insert({
+          name: 'Crazy',
+          artist_name: 'Patsy Cline',
+          genre: 'Country',
+          rating: 98
+        }, 'id')
+        .then(() => console.log('Seeding complete!'))
+        .catch(error => console.log(`Error seeding data: ${error}`))
+      ])
+    })
+    .catch(error => console.log(`Error seeding data: ${error}`));
+};
```

Comment:
This seed data is great. This will give us something to interact with before we have favorites from the musixmatch API!